### PR TITLE
[SHOT-4027] Wording tweaks and some data type exclusions added

### PR DESCRIPTION
**Pull request by rob-aitchison**

### CODE REVIEW

File: python/tk_alias/scene_data_validator.py [hunk 1]
```diff
@@ -172,7 +172,7 @@ def get_validation_data(self):
             "shader_unused": {
                 "name": "Delete Unused Shaders",
                 "description": "Check for shaders that are not assigned to any geometry.<br>"
-                               "The DefaultShader will be skipped.",
+                "The DefaultShader will be skipped.",
```
Comment (staceyoue) [inline python/tk_alias/scene_data_validator.py:175]: I think if you want to have it formatted as before to make it more readable, you can use triple quotes`"""`

**Review by staceyoue — APPROVED** [review]
LGTM!

File: python/tk_alias/scene_data_validator.py (hunk 1 above)
Comment (rob-aitchison) [reply python/tk_alias/scene_data_validator.py]: Thanks @staceyoue , that works so I updated them all now. :)

**Review by staceyoue — APPROVED** [review]
👍

**Pull request merged**